addons/mail/static/src/core/common/persona_model.js

```diff
@@ -15,6 +15,10 @@ export class Persona {
     localId;
     /** @type {number} */
     id;
+    /** @type {string} */
+    landlineNumber;
+    /** @type {string} */
+    mobileNumber;
```

Comment:
I don't see why those are in community, they are never returned here?

Reply:
It avoids creating a patch for those fields that, even if not used on the client side, exist in Community